exercises/series/src/example/java/Series.java

```diff
@@ -4,21 +4,20 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class Series {
+class Series {
 
     private final int digitsSize;
     private List<Integer> digits;
 
-    public Series(String string) {
-        this.digits = Arrays
-                .asList(string.split(("")))
-                .stream()
-                .map(digit -> Integer.parseInt(digit))
-                .collect(Collectors.toList());
+    Series(String string) {
+        this.digits =
+                Arrays.stream(string.split(("")))
```

Comment:
I would put this on the line above so that it's:
```
this.digits = Arrays.stream(string.split(""))
            .map(Integer::parseInt)
            .collect(Collectors.toList());
```
You might know this already but to make sure formatting is correct you can use ctrl + alt + L if you're using intellij on Windows (it's cmd + alt + L on mac). On eclipse it's ctrl + shift + F (cmd + shift + F on mac). I've found that very useful :)

Also `string.split((""))` can be `string.split("")`. Not your fault at all, I just didn't notice it until now :)